packages/firebase_data_connect/firebase_data_connect/lib/src/network/grpc_transport.dart

```diff
@@ -59,9 +63,11 @@ class GRPCTransport implements DataConnectTransport {
     } catch (e) {
       log('Unable to get app check token: $e');
     }
+    String sdkTypeValue =
+        "flutter/${sdkType == ClientSDKType.core ? 'core' : 'gen'}";
     Map<String, String> metadata = {
       'x-goog-request-params': 'location=${options.location}&frontend=data',
-      'x-goog-api-client': 'gl-dart/flutter fire/$packageVersion'
+      'x-goog-api-client': 'gl-dart/flutter fire/$packageVersion $sdkTypeValue'
```

Comment:
For our other platforms we use `gl-lang/`. Is the /flutter after gl-dart/ required? 

Example: https://github.com/firebase/firebase-js-sdk/pull/8480/files#diff-e6ca47cb91a2b2e0305e689ab4bad8faceff2a9370065604c953a838e3d69fabR27